plugins/cmd/chainlink-starknet/main.go

```diff
@@ -64,15 +36,15 @@ func main() {
 		HandshakeConfig: loop.PluginRelayerHandshakeConfig(),
 		Plugins: map[string]plugin.Plugin{
 			loop.PluginRelayerName: &loop.GRPCPluginRelayer{
-				PluginServer: cp,
+				PluginServer: p,
 				BrokerConfig: loop.BrokerConfig{
 					StopCh:   stopCh,
-					Logger:   lggr,
-					GRPCOpts: telem,
+					Logger:   s.Logger,
+					GRPCOpts: s.GRPCOpts,
 				},
 			},
 		},
-		GRPCServer: telem.NewServer,
+		GRPCServer: s.GRPCOpts.NewServer,
```

Comment:
`plugin.ServeConfig` has a `Logger` field too, but the default in this case is already what we need: Trace, StdErr, & JSON 
https://github.com/hashicorp/go-plugin/blob/main/server.go#L268-L271